### ISSUE TITLE
"Change 'share' link to format of [post-name](url)" doesn't seem to be working for the past few days

### ISSUE THREAD

**Issue by AmperAndSand** [soscripted/sox#432]
The "Change 'share' link to format of `[post-name](url)`" feature doesn't seem to be working for around two days now.

Steps to reproduce the behaviour:
1. Make sure "Change 'share' link to format of `[post-name](url)`" is enabled in SOX' settings
2. Go to any SE post, e.g. https://movies.stackexchange.com/questions/79161/how-did-they-make-cars-fall-apart-in-old-movies or https://gaming.stackexchange.com/questions/4999/how-can-i-tell-if-a-corpse-is-safe-to-eat
3. Click on 'Share'
4. Notice that the generated link is just the URL, and not in the `[post-name](URL)` format.

**SOX errors logged in console**
```
core.js:43 [Nano] Nano Defender Activated :: movies.stackexchange.com
chrome-extension://ggolfgbegefeeoocgjbmkembbncoadlb/content/rules-common.js:295 [Nano] Excluded :: uBO-Extra
userscript.html?id=3df8fa2b-61ef-489c-8a59-bef9307449e2:2240 soup injecting fixes
VM239:31 soup window.jQuery setter initialized
VM239:31 soup init complete
VM239:31 soup jQuery init fixes applied
VM239:31 soup setup complete
secure.quantserve.com/quant.js:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
sb.scorecardresearch.com/beacon.js:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: logging sox stored values --- 
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: access token set
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: SOX-apiCache
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: {"markEmployees":{"users":[{"is_employee":false,"creation_date":1560789408,"user_id":221020,"sox_request_time":1562897420737},{"is_employee":false,"creation_date":1338602034,"user_id":20726,"sox_request_time":1562897420737},{"is_employee":false,"creation_date":1308430921,"user_id":164007,"sox_request_time":1562897456425},{"is_employee":false,"creation_date":1302463615,"user_id":160166,"sox_request_time":1562897456425},{"is_employee":false,"creation_date":1294155657,"user_id":155278,"sox_request_time":1562897456425},{"is_employee":false,"creation_date":1275243752,"user_id":147532,"sox_request_time":1562897456425},{"is_employee":false,"creation_date":1260965995,"user_id":140478,"sox_request_time":1562897456425},{"is_employee":false,"creation_date":1246631594,"user_id":130301,"sox_request_time":1562897456425},{"is_employee":false,"creation_date":1562905317,"user_id":234442,"sox_request_time":1562913348669},{"is_employee":false,"creation_date":1562869682,"user_id":234422,"sox_request_time":1562913348669},{"is_employee":false,"creation_date":1562843616,"user_id":234398,"sox_request_time":1562913348669},{"is_employee":false,"creation_date":1562835623,"user_id":234389,"sox_request_time":1562913348669},{"is_employee":false,"creation_date":1562775475,"user_id":234345,"sox_request_time":1562913348669},{"is_employee":false,"creation_date":1562760284,"user_id":234337,"sox_request_time":1562913348669},{"is_employee":false,"creation_date":1562736967,"user_id":234327,"sox_request_time":1562913348669},{"is_employee":false,"creation_date":1560912287,"user_id":233285,"sox_request_time":1562913348669},{"is_employee":false,"creation_date":1507539557,"user_id":198086,"sox_request_time":1562913348669},{"is_employee":false,"creation_date":1407086807,"user_id":83800,"sox_request_time":1562913348669},{"is_employee":false,"creation_date":1438283568,"user_id":119944,"sox_request_time":1562921240898},{"is_employee":false,"creation_date":1436021944,"user_id":117494,"sox_request_time":1562921240898},{"is_employee":false,"creation_date":1421655279,"user_id":99988,"sox_request_time":1562921240898},{"is_employee":false,"creation_date":1462462868,"user_id":146705,"sox_request_time":1562921408958},{"is_employee":false,"creation_date":1458469524,"user_id":142490,"sox_request_time":1562921408958},{"is_employee":false,"creation_date":1406875146,"user_id":83632,"sox_request_time":1562921408958},{"is_employee":false,"creation_date":1391351647,"user_id":68284,"sox_request_time":1562921408958},{"is_employee":false,"creation_date":1342464991,"user_id":29425,"sox_request_time":1562921408958},{"is_employee":false,"creation_date":1304170818,"user_id":9154,"sox_request_time":1562921408958},{"is_employee":false,"creation_date":1502413052,"user_id":194793,"sox_request_time":1562921793756},{"is_employee":false,"creation_date":1501183452,"user_id":193861,"sox_request_time":1562921798673},{"is_employee":false,"creation_date":1498749531,"user_id":190055,"sox_request_time":1562921798673},{"is_employee":false,"creation_date":1497374659,"user_id":188991,"sox_request_time":1562921798673},{"is_employee":false,"creation_date":1495801668,"user_id":187811,"sox_request_time":1562921798673},{"is_employee":false,"creation_date":1469523731,"user_id":158353,"sox_request_time":1562921798673},{"is_employee":false,"creation_date":1468310485,"user_id":154626,"sox_request_time":1562921798673},{"is_employee":false,"creation_date":1457095111,"user_id":140943,"sox_request_time":1562921798673},{"is_employee":false,"creation_date":1404384848,"user_id":80847,"sox_request_time":1562921798673},{"is_employee":false,"creation_date":1445052638,"user_id":127413,"sox_request_time":1562923962473},{"is_employee":false,"creation_date":1428160214,"user_id":107390,"sox_request_time":1562923962473},{"is_employee":false,"creation_date":1444850334,"user_id":127239,"sox_request_time":1562925202643},{"is_employee":false,"creation_date":1435424969,"user_id":116820,"sox_request_time":1562925202643},{"is_employee":false,"creation_date":1334594260,"user_id":23986,"sox_request_time":1562925601216},{"is_employee":false,"creation_date":1405200984,"user_id":82435,"sox_request_time":1562934649998},{"is_employee":false,"creation_date":1560276308,"user_id":232912,"sox_request_time":1562935271426},{"is_employee":false,"creation_date":1453516445,"user_id":136881,"sox_request_time":1562935271426},{"is_employee":false,"creation_date":1453383973,"user_id":136727,"sox_request_time":1562935271426},{"is_employee":false,"creation_date":1334652580,"user_id":24033,"sox_request_time":1562935271426},{"is_employee":false,"creation_date":1544193890,"user_id":222379,"sox_request_time":1562995791788},{"is_employee":false,"creation_date":1522525600,"user_id":209174,"sox_request_time":1562995791788},{"is_employee":false,"creation_date":1562983704,"user_id":234493,"sox_request_time":1562995857225},{"is_employee":false,"creation_date":1481848885,"user_id":174197,"sox_request_time":1562997070050},{"is_employee":false,"creation_date
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: SOX-lastVersionInstalled
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: 2.5.25DEV
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: SOXSETTINGS
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: ["Appearance-addAuthorNameToInboxNotifications","Appearance-answerTagsSearch","Appearance-colorAnswerer","Appearance-highlightQuestions","Appearance-isQuestionHot","Appearance-localTimestamps","Appearance-markEmployees","Appearance-metaChatBlogStackExchangeButton","Appearance-metaNewQuestionAlert","Appearance-scrollToTop","Appearance-standOutDupeCloseMigrated","Appearance-tabularReviewerStats","Appearance-unspoil","Appearance-addTimelineAndRevisionLinks","Comments-autoShowCommentImages","Comments-commentReplies","Comments-commentShortcuts","Comments-copyCommentsLink","Comments-showCommentScores","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-kbdAndBullets","Editing-titleEditDiff","Flags-flagOutcomeTime","Flags-flagPercentages","Sidebar-linkedToFrom","Extras-linkedPostsInline","Extras-parseCrossSiteLinks","Extras-quickAuthorInfo","Extras-shareLinksMarkdown","Extras-warnNotLoggedIn"]
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: editReasons
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: [{"formatting":"Improved Formatting"},{"spelling":"Corrected Spelling"},{"grammar":"Fixed grammar"},{"greetings":"Removed thanks/greetings"}]
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: metaNewQuestionAlert-lastQuestions
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: {"meta.gaming":"Why we should make a flag reason for closing modded Minecraft tech support questions","meta.movies":"Narrative questions related to TV shows / films","meta.superuser":"Why was my flag to stop my bounty rejected?","meta.gaming.stackexchange":"My game identification question was closed as off-topic. Where can I ask for help instead?","meta.worldbuilding.stackexchange":"Sandbox for Proposed Questions"}
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: SOX-debug
userscript.html?id=5cc3be1c-f583-4c68-9516-ece226528f4d:68 SOX: true
VM239:31 soup JS fixes applied
VM239:31 soup subscribed to realtime feed for question 79161 on site 367
```

**Expected behaviour**
The generated 'share' link should be in the  `[post-name](url)` format.

**Environment**
SOX version: 2.5.25DEV
Platform: Tampermonkey

---

### Features Enabled 

    ["Appearance-addAuthorNameToInboxNotifications","Appearance-answerTagsSearch","Appearance-colorAnswerer","Appearance-highlightQuestions","Appearance-isQuestionHot","Appearance-localTimestamps","Appearance-markEmployees","Appearance-metaChatBlogStackExchangeButton","Appearance-metaNewQuestionAlert","Appearance-scrollToTop","Appearance-standOutDupeCloseMigrated","Appearance-tabularReviewerStats","Appearance-unspoil","Appearance-addTimelineAndRevisionLinks","Comments-autoShowCommentImages","Comments-commentReplies","Comments-commentShortcuts","Comments-copyCommentsLink","Comments-showCommentScores","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-kbdAndBullets","Editing-titleEditDiff","Flags-flagOutcomeTime","Flags-flagPercentages","Sidebar-linkedToFrom","Extras-linkedPostsInline","Extras-parseCrossSiteLinks","Extras-quickAuthorInfo","Extras-shareLinksMarkdown","Extras-warnNotLoggedIn"]

**Comment by double-beep** [comment]
See #431 which should fix that.

**Comment by shu8** [comment]
@double-beep's fix has been merged in [dev 2.5.31](https://github.com/soscripted/sox/raw/dev/sox.user.js) :)
